### ISSUE TITLE
Hillshading artefacts: terrain below 0 NAP and just over NL country-borders

### ISSUE THREAD

**Issue by niekvanruler** [map5nl/map5#38]
Wat zijn de gerafelde vormen in onderstaand scrrenshot?
Locatie: https://app.map5.nl/map5topo/sidebyside/#map=11.00/230978.48/577747.10/0/4

![image](https://github.com/map5nl/map5issues/assets/10760567/ee4e6f60-038c-4377-a448-d1d5e7166edc)

**Comment by justb4** [comment]
@niekvanruler we zijn bekend met dit verschijnsel. Gebeurt op aantal plekken. We hadden intern al een ticket openstaan. 
Het heeft te maken met de "hillshading" die we doen vanuit de AHN3. Ope map5topo_simple zul je dit niet zien.
De hillshading levert op enkele plekken, vooral waar NL rond 0 NAP gaat, dergelijke artefacten. Denk dat het iets met "gaten" in AHN3 te maken heeft.

We gaan binnenkort sowieso over op AHN4. Hopelijk is dit dan verdwenen, maar staat op onze radar.

**Comment by niekvanruler** [comment]
Bij de grens vanaf zoomlevel 7 en hoger zien we hetzelfde effect.
Maar daar komt het denk ik niet door dezelfde oorzaak als hierboven?
![image](https://github.com/map5nl/map5issues/assets/10760567/2de80f0d-803a-4247-b52d-fb204c6d9ad9)

**Comment by justb4** [comment]
Ja, is zelfde effect net buiten de grenzen, de AHN houdt daar op, gaat naar 0m, maar met artefact, zowel met 5m als 0.5m.
Probleem is intern bekend, oplossing wordt gezocht, mogelijk een "cutline" op de GeoTIFFs met NL grenzen geometrie.

**Comment by justb4** [comment]
In Release 2023-09 zullen de artefacten een stuk minder worden, de hillshade voor heel NL opnieuw gegenereerd, nog niet geheel weg:

![image](https://github.com/map5nl/map5issues/assets/582630/83f4ef7c-25e5-45ef-b288-3455d818d316)

**Comment by niekvanruler** [comment]
Wat mij betreft kan deze dicht, als jullie toch al een ticket hebben openstaan.

**Comment by justb4** [comment]
Laat deze maar open: de ticket die we hebben is intern. De ticket hier is zichtbaar voor iedereen dus als geheel opgelost, gaan we sluiten. Heb ook de titel veranderd om issue duidelijker/breder te maken.
